guides/micronaut-oracle-autonomous-db/micronaut-oracle-autonomous-db.adoc

```diff
@@ -8,7 +8,8 @@ To complete this guide, you will need the following:
 * A decent text editor or IDE
 * JDK 11 or greater installed with `JAVA_HOME` configured appropriately
 * An Oracle Cloud account (create a free trial account at https://signup.oraclecloud.com[signup.oraclecloud.com])
-* https://docs.cloud.oracle.com/en-us/iaas/Content/API/SDKDocs/cliinstall.htm[Oracle Cloud CLI] installed, with local access to Oracle Cloud configured by running `oci setup config`
+* https://docs.cloud.oracle.com/en-us/iaas/Content/API/SDKDocs/cliinstall.htm[Oracle Cloud CLI] installed, with local access to Oracle Cloud configured by running `oci setup config`. More about the configuration you can find at https://micronaut-projects.github.io/micronaut-oracle-cloud/snapshot/guide/[Micronaut Oracle cloud].
```

Comment:
I wouldn't mix the requirements with micronaut docs. The truth is that currently the installation of `oci` on Macs takes really long time, and the cli itselve is not genuinely required to finish the guideline. It would be more convenient to link https://docs.oracle.com/en-us/iaas/Content/API/Concepts/sdkconfig.htm#SDK_and_CLI_Configuration_File saying like it's preferable option to create the config file manually or download it directly if possible (the screenshot would be ok here I guess)

Reply:
Besides what Palo said, please don't link to snapshot docs, always use `latest`

Reply:
Does this description sound good to you?
```
* Oracle Cloud SDK configured. It's a preferable option to create the config file https://docs.oracle.com/en-us/iaas/Content/API/Concepts/sdkconfig.htm#SDK_and_CLI_Configuration_File[manually] or to download it from the Oracle Cloud if possible. Another easier way is to get https://docs.cloud.oracle.com/en-us/iaas/Content/API/SDKDocs/cliinstall.htm[Oracle Cloud CLI] installed, with local access to Oracle Cloud configured by running `oci setup config`.
```
I am not sure about the screenshot what should be on the image and where shall I put it?